resources/pods/cmk-isolate-pod.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
 # Consumed by mutating webhook
   annotations:
     cmk.intel.com/mutate: "true" # accepted values to trigger mutation: "true", "True", "1"
-  #namespace: user-supplied-namespace
+    namespace: cmk-namespace
```

Comment:
Looks like the extra space here stops the isolate pod running the cmk-namespace

Reply:
Yup. Fixed it.